install-git-secrets-hook.sh

```diff
@@ -40,8 +40,9 @@ fi
 git secrets --register-aws $GLOBAL_TOKEN
 
 # Add general custom rules
-git secrets --add $GLOBAL_TOKEN '(dbpasswd|dbuser|dbname|dbhost|api_key|apikey|password|user|guid|hostname|pw|auth)\s*[=:]\s*['"'"'0-9a-zA-Z_\/+!{}=-]{4,120}'
-git secrets --add $GLOBAL_TOKEN '(DBPASSWD|DBUSER|DBNAME|DBHOST|API_KEY|APIKEY|PASSWORD|USER|GUID|HOSTNAME|PW|AUTH)\s*[=:]\s*['"'"'0-9a-zA-Z_\/+!{}=-]{4,120}'
```

Comment:
why do we want to exclude html?

Reply:
There were a lot of false positives in the html code, though it was mainly with "user" and "auth."  My thought was that the only thing we're really concerned with is assignment; we don't want someone submitting password="TheActualPassword" or something along that line.  I can't think of any context in which that would be done in html code.